ncbi_genome_download/core.py

```diff
@@ -174,7 +190,11 @@ def download(**kwargs):
     taxid = kwargs.pop('taxid', EDefaults.TAXID.default)
     human_readable = kwargs.pop('human_readable', False)
     parallel = kwargs.pop('parallel', EDefaults.PROCESSES.default)
-    # TODO: warning/error if unrecognized option
+    # FIXME: improve error handling and feedback
+    assert len(kwargs) == 0, "Unrecognized option(s)"
+    if len(kwargs) > 0:
```

Comment:
That code is unreachable, as the line above already exits if `len(kwargs) != 0`.